api/envoy/extensions/wasm/v3/wasm.proto

```diff
@@ -37,6 +37,8 @@ message VmConfig {
   config.core.v3.AsyncDataSource code = 3;
 
   // The Wasm configuration used in initialization of a new VM (proxy_on_start).
+  // `google.protobuf.Struct` and `google.protobuf.Value` are serialized as JSON before passing them
```

Comment:
Please see my comment in [#10781](https://github.com/envoyproxy/envoy/issues/10781#issuecomment-613916342)

Reply:
Are there any follow ups here? Do we want to specialize further types or just leave it at this for now?

/wait-any